content/essentials/signals.mdx

```diff
@@ -0,0 +1,105 @@
+import { Aside } from "~/components/configurable/Aside";
+
+# Signals
+
+At the heart of a Solid's reactivity system are signals, acting as the building blocks for tracking, updating, and responding to changes in data. 
+When the information in a signal changes, the parts of your application that dependent on that signal are automatically adjusted to represent it.
+
+Signals eliminate the need to re-execute the entire component function, focusing instead on updating only the altered aspects of the component. 
+Furthermore, this adaptability of signals extends beyond component functions, allowing them to be used in other contexts and scenarios within the application. 
+
+## Creating a signal
+
+To create a signal, import the `createSignal`  primitive from Solid at the top of the file:
+
+```jsx
+import { createSignal } form "solid-js";
+```
+Upon its call, this function yields a pair of values: the first is a **getter **function, which is responsible for _accessing _the current value of the signal, and the second is a **setter **function, designed to _update _the signal's value:
+
+```jsx
+const [count, setCount] = createSignal(0);
+```
+
+In this example, `count` is the getter and `setCount`  is the setter function.
+
+<Aside>
+
+The syntax using `[` and `]` is called [array destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment). 
+This lets you extract values from the array. 
+Specifically, in the context of `createSignal` , there are two items being returned: the getter (first), and the setter (second).
+
+</Aside>
+
+Signals allow you to manage a variety of data types, including both objects and arrays. 
```

Comment:
This is worded confusingly. Signals are just reactive variables, like static values they can simply point to anything. Saying that they can manage objects and arrays explicit though makes me think that I should put objects and arrays inside a signal, when that's often not the case, because one often wants fine-grained updates for objects and arrays, which a single signal can't do.